src/Illuminate/Database/Eloquent/Concerns/HasRelationships.php

```diff
@@ -63,6 +65,26 @@ public function hasOne($related, $foreignKey = null, $localKey = null)
     }
 
     /**
+     * Define a one-to-one relationship that goes through an intermediate model.
+     *
+     * @param string      $related           Class name of related Model
```

Comment:
Please remove all comments from this part of the phpdoc, we don't use this style. If anything is unclear, add a note in the "long description" part of the phpdoc.